build.sbt

```diff
@@ -82,6 +84,22 @@ lazy val bundleZioHttp4sBlaze = project
     )
   )
 
+lazy val datastaxJavaDriver = project
+  .in(file("datastax-java-driver"))
+  .settings(commonSettings)
+  .settings(
+    name := "sst-datastax-java-driver",
```

Comment:
Sorry to be a bit late to the party, but do you really think naming the module like this is a good idea?

"Java" is completely redundant. "Driver" for the most part as well, especialy if the name contained something more descriptive. Leading the name with "Datastax" which is a name of company, not a project also does not feel right. And I miss the most important information in the name, that it is Cassandra driver.

I understand that it is aligned with the name of its dependency, but that pattern is not even unconditionaly enforced in other modules.

So I propose to change it to one of the following (in order from the best): 1. cassandraDatastax 2. cassandraDatastaxDriver 3. datastaxDriver

Reply:
I've missed this comment before. Can you please point out where this convention does not hold in other modules? As far as I know it should hold for all the modules and even though I understand your point and I am not strongly against it I like when things follow conventions and in this case when you write "datastax java driver" into Google you'll get what you want as the first result. So it seems to me it works quite well.

Reply:
Those inconsistencies are e.g. "micrometerJmx" module (io.micrometer:micrometer-registry-jmx) omits "registry" from artifactId. So I think this module could omit "java" as well.
Also other modules do not use group id in their name, e.g. "doobieHikari" module (org.tpolecat:doobie-hikari) is not named tpolecatDoobieHikari.
I acknowledge the googling argument, even "java cassandra driver" gets you this one.
In the end, I guess this is minor issue and I don't want to block the merge, so I am fine with whatever you decide.

Reply:
Alright, so I think `cassandraDatastaxDriver` would be acceptable for me.

Reply:
So `CassandraDatastaxDriver` it is.